src/index.css

```diff
@@ -16,6 +16,10 @@ html {
   color: #47566B;
 }
 
+a {
+  color: #249bbb;
```

Comment:
We would like to avoid adding universal styles to this repo, can you refactor and add specificity?

Reply:
I actually know why he did this. Currently, our hyperlinks aren't showing up as a different color, but this isn't the fix we need. We need to create an issue to fix the stylesheets from https://github.com/OperationCode/operationcode_frontend/pull/451

Reply:
Done - made #555 

This line should be reverted regardless.